AuxTel/Troubleshooting/General-Troubleshooting/AuxTel-Elevation-out-of-range.rst

```diff
@@ -0,0 +1,124 @@
+.. Review the README in this procedure's directory on instructions to contribute.
+.. Static objects, such as figures, should be stored in the _static directory. Review the _static/README in this procedure's directory on instructions to contribute.
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *I. Sotuela*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Carlos Morales, Erik Dennihy*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _AuxTel-AuxTel-Troubleshooting-General-Troubleshooting-AuxTel-AuxTel-Elevation-out-of-range:
+.. Each section should include a label for cross-referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with a reST object such as a title or figure, you must include the link with an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+######################################
+AuxTel Elevation Out of range
+######################################
+
+.. _AuxTel-Elevation-out-of-range-Overview:
+
+Overview
+========
+
+.. In one or two sentences, explain when this AT correct_pointing failure procedure needs to be used. Describe the symptoms that the user sees to use this procedure.
+
+This procedure should be used when the AuxTel :file:`auxtel/correct_pointing.py` script fails with the error *Rejected: elevation out-of-range*.
```

Comment:
Error should be in code-block

Reply:
or in " marks (confirm with Iona) because italics is used for specific situations

Reply:
italics are used specifically in pathnames, filenames etc. You could use boldface to highlight anuthing else. Refer to the Rest Guide on confluence and confirm with @isotuela

Reply:
Ok, changed it to code-block. Thanks!

Reply:
Thanks Ioana !